### PR TITLE
Add 20+ parameters for NATS source doc

### LINKED ISSUE #2686: Document: feat(source): add NATS source consumer parameters
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/17615

### Which part(s) of the docs might be affected or should be updated? And how?

https://docs.risingwave.com/docs/current/ingest-from-nats/
Add missing parameters

### Reference

_No response_

### PR DESCRIPTION
**Pull request by WanYixian**
## Description

Add missing parameters for NATS source and one example

## Related code PR

https://github.com/risingwavelabs/risingwave/pull/17615

## Related doc issue

Resolve https://github.com/risingwavelabs/risingwave-docs/issues/2686

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/ingest/ingest-from-nats.md [hunk 1]
```diff
@@ -98,6 +98,29 @@ According to the [NATS documentation](https://docs.nats.io/running-a-nats-servic
...
+| `consumer.name` | Optional. Name of the consumer. |
+| `consumer.description` | Optional. Description of the consumer. |
+| `consumer.deliver_policy` | Optional. Policy on how messages are delivered. |
+| `consumer.ack_policy` | Optional. Acknowledgment policy for message processing (e.g., `None`, `All`, `Explicit`). |
+| `consumer.ack_wait.sec` | Optional. Time to wait for acknowledgment before considering a message as undelivered. |
+| `consumer.max_deliver` | Optional. Maximum number of times a message will be delivered. |
+| `consumer.filter_subject` | Optional. Filter for subjects that the consumer will process. |
+| `consumer.filter_subjects` | Optional. List of subjects that the consumer will filter on. |
+| `consumer.replay_policy` | Optional. Policy for replaying messages (e.g., `Instant`, `Original`). |
+| `consumer.rate_limit` | Optional. Rate limit for message delivery in bytes per second. |
```
Comment (emile-00) [inline docs/ingest/ingest-from-nats.md]: In https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/struct.Config.html `rate_limit` is defined in bits per second

File: docs/ingest/ingest-from-nats.md [hunk 2]
```diff
@@ -98,6 +98,29 @@ According to the [NATS documentation](https://docs.nats.io/running-a-nats-servic
...
+| `consumer.sample_frequency` | Optional. Frequency for sampling messages. |
```
Comment (emile-00) [inline docs/ingest/ingest-from-nats.md]: Can add that it's between 0-100 if it's the same as what's outlined in [docs](https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/struct.Config.html)

File: docs/ingest/ingest-from-nats.md [hunk 3]
```diff
@@ -120,3 +143,38 @@ WITH
   );
 ```
 
+The parameters supported by the `async_nats` crate are all supported in the RisingWave NATS source connector.
```
Comment (emile-00) [inline docs/ingest/ingest-from-nats.md]: Maybe can link `async_nats` to their [docs](https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/struct.Config.html)?

**Review by emile-00 — APPROVED** [review]
Added a few minor comments, otherwise LGTM!

**Review by tabVersion — APPROVED** [review]
rest LGTM

File: docs/ingest/ingest-from-nats.md (hunk 2 above)
Comment (WanYixian) [reply docs/ingest/ingest-from-nats.md]: Thanks for pointing out. 
Hi @tabVersion, the `sample_frequency` is a percentage in the [async-nats docs](https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/struct.Config.html), but here in our example it's not. Is it on purpose? 🤔 
image
image

Comment (WanYixian) [reply docs/ingest/ingest-from-nats.md]: As talked offline, edited the example and updated the doc to be 1-100

**Pull request merged**